src/assets/locales/fr.json

```diff
@@ -136,7 +159,10 @@
   "exited-room-screen.reason.left": "Vous avez quitté la salle.",
   "exited-room-screen.reason.scene-error": "Le chargement de la scène a échoué.",
   "exited-room-screen.reason.version-mismatch": "La version que vous avez déployée n'est pas encore disponible. Votre navigateur va se rafraîchir dans 5 secondes.",
+  "exited-room-screen.refresh-page-button": "Rafraichir la page",
   "exited-room-screen.source-link": "Si vous souhaitez utiliser votre propre serveur, le code source de Hubs est disponible sur <a>GitHub</a>.",
+  "footer.FAQ": "FAQ",
+  "footer.hubs-merch": "Marchandises Hubs",
```

Comment:
Suggestion : "La boutique Hubs officielle"

Reply:
I changed with just "Boutique Hubs". It's a link in the footer, there is already lots of links in the footer, shorter the better.